packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -850,6 +850,36 @@ const replaceNodeHtmlLinks = ({ wpUrl, nodeString, node }) => {
   return nodeString
 }
 
+// replaces specific string or regex with a given string from the plugin options config
+const replaceNodeHtmlStrings = ({ nodeString, node, pluginOptions }) => {
+  if (Array.isArray(pluginOptions?.searchAndReplace)) {
+    pluginOptions.searchAndReplace.forEach(({ search, replace }) => {
```

Comment:
A test for this would be nice :)

Reply:
I  wrote a test, but I couldn't run it easily, I ran Jest (`yarn jest` or `yarn jest packages/gatsby-source-wordpress`) but somehow some dependency wasn't met. So I pushed the test blindly hopping the CI could give me a positive result :crossed_fingers: